build.gradle

```diff
@@ -45,3 +45,19 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+
+configurations {
+    ktlint
+}
+
+dependencies {
+    ktlint "com.github.shyiko:ktlint:0.14.0"
+}
+
+task ktlint(type: JavaExec, group: "verification") {
+    description = "Check Kotlin code style."
+    classpath = configurations.ktlint
+    main = "com.github.shyiko.ktlint.Main"
+    args "**/*.kt"
```

Comment:
The docs also include:
```
    // to generate report in checkstyle format prepend following args:
    // "--reporter=plain", "--reporter=checkstyle,output=${buildDir}/ktlint.xml"
    // see https://github.com/shyiko/ktlint#usage for more
```

I wonder if this is desirable so if we ever parse checkstyle results, we can just re-use that code.